pos-print/build.gradle

```diff
@@ -61,7 +60,7 @@ dependencies {
   compile('com.google.sitebricks:sitebricks-client:0.8.11') {
     exclude module: 'guice'
     exclude module: 'guice-servlet'
-    exclude module: 'jackson-databind'
+//    exclude module: 'jackson-databind'
```

Comment:
The commented line need to be removed if jackson is no longer going to be ignored.